cnames_active.js

```diff
@@ -2772,7 +2772,7 @@ var cnames_active = {
   "sociably": "machinat.github.io/sociably",
   "social-likes": "sapegin.github.io/social-likes", // noCF? (don´t add this in a new PR)
   "social-likes-next": "sapegin.github.io/social-likes-next", // noCF? (don´t add this in a new PR)
-  "solace": "solacejs.github.io/solace.js",
+  "solace": "cname.vercel-dns.com",
```

Comment:
```suggestion
  "solace": "cname.vercel-dns.com", // noCF
```
As the CI error notes, you'll want to disable the Cloudflare proxy when using Vercel